admin/sales_analytics/utils.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from admin.base.settings import ENTRY_POINTS
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+# Helper functions
+def get_entry_point(system_tags, entry_points=ENTRY_POINTS):
+    """
+    Given the user system_tags, return the user entry point (osf, osf4m, prereg, institution)
+    In case of multiple entry_points existing in the system_tags, return only the first one.
+    """
+    entry_points = entry_points.keys()
+    for i in system_tags:
```

Comment:
This loop makes a few assumptions: That there is only one system tag from the `entry_points` and that it will be in order.

Also it is standard practice to only use `i` as an integer variable in a loop. Please consider using `tag` or some other somewhat descriptive name.